### PR TITLE
The vertical header should stay to the left of the card body when in mobile and tablet view.

### PR DESCRIPTION
**Pull request by julieduong22**
The vertical header should stay to the left of the card body when in mobile and tablet view.

### CODE REVIEW

File: src/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -191,8 +191,12 @@ $inner-border-radius: calc(#{$sds-card-border-radius} - #{$sds-card-border-width
   &--vertical {
 
     @include at-media('mobile') {
+      @include u-display('flex');
+
       .sds-card__header {
-        text-align: center;
+        align-items: center;
+        justify-content: left;
+        text-align: left;
```
Comment (davereed) [inline src/stylesheets/components/_card.scss:199]: We should retain the border right that is currently only at the tablet level. Move that to mobile and larger size.

File: src/stylesheets/components/_card.scss [hunk 2]
```diff
@@ -191,8 +191,12 @@ $inner-border-radius: calc(#{$sds-card-border-radius} - #{$sds-card-border-width
   &--vertical {
 
     @include at-media('mobile') {
```
Comment (davereed) [inline src/stylesheets/components/_card.scss:193]: Do we still need mobile and tablet breakpoint if all sizes should have the header on the left?

**Comment by davereed** [comment]
- Don't commit package.json and package-lock.json when doing a feature branch (unless the branch is intended to update the dependencies).
- For pull requests, use a more descriptive name.
- See inline code comments.

**Review by davereed — CHANGES_REQUESTED** [review]

File: src/stylesheets/components/_card.scss (hunk 1 above)
Comment (julieduong22) [reply src/stylesheets/components/_card.scss:199]: got it.

File: src/stylesheets/components/_card.scss (hunk 2 above)
Comment (julieduong22) [reply src/stylesheets/components/_card.scss:193]: We can keep the tablet breakpoint in case you want any changes for tablet.

**Review by davereed — APPROVED** [review]

**Pull request merged**